types/google.analytics/index.d.ts

```diff
@@ -524,7 +525,7 @@ declare namespace UniversalAnalytics {
         socialNetwork?: string;
         socialTarget?: string;
         some?: string;
-        step?: boolean;
+        step?: number;
```

Comment:
For backwards-compatibility, should this be a union of number and boolean instead?